ci/maketmpl/md_image.go

```diff
@@ -0,0 +1,76 @@
+package maketmpl
+
+import (
+	"io"
+	"log"
+	"strings"
+
+	htmlx "golang.org/x/net/html"
+)
+
+type mdImage struct {
+	start int
+	end   int
+
+	src string
+	alt string
+
+	heightPx string
+	widthPx  string
+
+	style string
+
+	isHTML bool
```

Comment:
seems `isHTML` always true? can be removed?

Reply:
it is used later. non-html / md image tags are directly parsable with regex, so it is much simpler.. but let me add it in here. give me a minute.

Reply:
added the part that is parsing images from entire md file now.

Reply:
and now the field is used :)